Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2142,6 +2245,55 @@ var stagger = function(
   }));
 };
 
+type LoopAnimationConfig = AnimationConfig & { iterations: number };
```

Comment:
I don't think we want any props from AnimationConfig, the type should just be `{ iterations: number }`.